### ISSUE TITLE
How are classes indexed w.r.t F1 metrics when used with a MultiClass classifier?

### ISSUE THREAD

**Issue by JeroenSchmidt** [catboost/catboost#1458]
Problem: 
How do I link the raw labels to the indices used by the Multiclass F1 scores produced by the model?

Are the class indices generated according to the order in which the labels appear in the `class_names` veriable?

Example code:
```
class_names = ["A","B","C"]
model = CatBoostClassifier(
        loss_function='MultiClass',
        custom_metric=["F1"],
        eval_metric="TotalF1",
        random_seed=42,
        logging_level='Silent',
        class_names=class_names
)

model.fit(X,y, cat_features=categorical_features_indices)
model.get_best_score().get("learn").keys()
>> F1:class=2, F1:class=0, F1:class=1 
## Am I correct to assume that because of the order of  `class_names ` then the mapping is as follows?
Class A -> Class 0
Class B -> Class 1
Class C -> Class 2
```

catboost version: 0.23.2
Operating System: MacOS 10.15

**Comment by nikitxskv** [comment]
Hi @JeroenSchmidt !
You can use `classes_` attribute (https://catboost.ai/docs/concepts/python-reference_catboost_attributes.html#classes) of trained model to find out the order of classes in resulting predictions.
